test/unit/client-snapshot-agent-serviceaccount.bats

```diff
@@ -4,17 +4,13 @@ load _helpers
 
 @test "client/SnapshotAgentServiceAccount: disabled by default" {
   cd `chart_dir`
-  local actual=$(helm template \
-      -x templates/client-snapshot-agent-serviceaccount.yaml  \
-      . | tee /dev/stderr |
-      yq 'length > 0' | tee /dev/stderr)
-  [ "${actual}" = "false" ]
+  assert_empty helm template -s templates/client-snapshot-agent-serviceaccount.yaml .
```

Comment:
wow! kind of cool that it became a one-liner 😄

Reply:
haha yes!